tests/test_backend_selector.py

```diff
@@ -0,0 +1,104 @@
+import os
+import graph_tool
+import numpy as np
+import pandas as pd
+import argparse
+import graph_tool.all as gt
+import networkx as nx
+from scipy.sparse import coo_matrix
+
+from pp_netlib.network import Network
+
+def get_args():
+    parser = argparse.ArgumentParser(description="Testing backend selector.")
+    parser.add_argument("--set_with_python", action="store_true", help="Use flag to set backend with od.environ.")
+    args = parser.parse_args()
+    set_with_py = args.set_with_python
+    return set_with_py
+
+def __init__():
+    vertex_labels = ["s1", "s2", "s3", "s4", "s5"]
+    sample_edge_list = [(0, 1), (1, 2), (2, 3), (3, 4), (4, 0)]
+    weights = [0.1, 0.1, 0.5, 0.7, 0.2]
+    sample_weighted_edge_list = [(0, 1, 0.1), (1, 2, 0.1), (2, 3, 0.5), (3, 4, 0.7), (4, 0, 0.2)]
```

Comment:
Did you fix the issue from before with missing some of the edges? It looks like the edge list here is a lot smaller than before

Reply:
yep, that issue is fixed, but it is in a different branch. should i make those changes here?

Reply:
Yes, make them here

Reply:
added those changes